lib/model/CResourceMonitor.cc

```diff
@@ -45,20 +44,22 @@ void CResourceMonitor::memoryUsageReporter(const TMemoryUsageReporterFunc& repor
     m_MemoryUsageReporter = reporter;
 }
 
-void CResourceMonitor::registerComponent(CAnomalyDetector& detector) {
-    LOG_TRACE(<< "Registering component: " << &detector);
-    m_Detectors.emplace(&detector, std::size_t(0));
+void CResourceMonitor::registerComponent(CMonitoredResource& resource) {
+    LOG_TRACE(<< "Registering component: " << &resource);
+    m_Resources.emplace(&resource, std::size_t(0));
 }
 
-void CResourceMonitor::unRegisterComponent(CAnomalyDetector& detector) {
-    auto itr = m_Detectors.find(&detector);
-    if (itr == m_Detectors.end()) {
-        LOG_ERROR(<< "Inconsistency - component has not been registered: " << &detector);
+void CResourceMonitor::unRegisterComponent(CMonitoredResource& resource) {
+    auto itr = m_Resources.find(&resource);
+    if (itr == m_Resources.end()) {
+        LOG_ERROR(<< "Inconsistency - component has not been registered: " << &resource);
```

Comment:
```suggestion
        LOG_ERROR(<< "Inconsistency. - Cannot unregister a component that has not yet been registered: " << &resource);
```

Reply:
Reformatting like this would make the message inconsistent with all the other similar ones - `grep Inconsistency lib/*/*.cc` and you'll see.  Also, these are basically soft asserts, i.e. things that should never happen but if they did we decided not to abort the program but let it carry on.  So hopefully nobody will ever see such a message logged and the exact wording is not particularly important.